style/tree/tree.css

```diff
@@ -0,0 +1,124 @@
+/********************************************************************************
+ * Copyright (C) 2024 Arm Limited and others.

```

Comment:
Probably also add EclipseSource. Don't recognize a lot anymore from the original file.